predicators/envs/spot_env.py

```diff
@@ -1811,7 +1829,8 @@ def _generate_goal_description(self) -> GoalDescription:
     def _run_init_search_for_objects(
         self, detection_ids: Set[ObjectDetectionID]
     ) -> Dict[ObjectDetectionID, math_helpers.SE3Pose]:
-        """Override to have the hand look down at the table at first."""
+        """Override to move the hand up so it can easily see when the ball is
```

Comment:
did you mean to change this docstring instead of the other env?

Reply:
nope - whoops!